### PR TITLE
fix: add /health endpoint

### PR DESCRIPTION
**Pull request by nanikjava**
PR Fix : #869 

Add `/health` endpoint for health check. 

The output will be

```
{"status":"up"}
```

To test with `curl`

`curl localhost:8081/health`

### CODE REVIEW

**Comment by nanikjava** [comment]
@buixor WIP PR is up. Let me know what you think about this, still rough want to show the idea first.

File: pkg/apiserver/controllers/controller.go
Comment (buixor) [inline pkg/apiserver/controllers/controller.go:49]: I think there is unwanted `q` here

**Comment by buixor** [comment]
Hello ! I think it looks like a simple but sufficient approach :+1: 
I'm off next week, but we can think about if further checks are needed or if it might be enough for a first iteration !

File: pkg/apiserver/controllers/controller.go
Comment (nanikjava) [reply pkg/apiserver/controllers/controller.go]: Removed

**Comment by nanikjava** [comment]
> Hello ! I think it looks like a simple but sufficient approach 
> I'm off next week, but we can think about if further checks are needed or if it might be enough for a first iteration !

@buixor thanks...PR updated with bit of clean up

**Review by buixor — APPROVED** [review]
lgtm

**Comment by buixor** [comment]
thanks, looks good.
let's wait for a second review and merge, should make its way in the next release :+1:

**Review by sabban — APPROVED** [review]
lgtm

Just one thought. This adds a dependency, I assume it's worthy.

**Pull request merged**
